cli/internal/login/link.go

```diff
@@ -237,6 +233,44 @@ func promptSetup(location string) (bool, error) {
 	return shouldSetup, nil
 }
 
+func addTurboToGitignore() error {
+	fs.EnsureDir(".gitignore")
+	gitignoreBytes, err := os.ReadFile(".gitignore")
+	if err != nil {
+		return fmt.Errorf("could not find or update .gitignore.\n%w", err)
+	}
+
+	hasTurbo := false
+	gitignoreContents := string(gitignoreBytes)
+	gitignoreLines := strings.Split(gitignoreContents, "\n")
+
+	for _, line := range gitignoreLines {
+		if strings.TrimSpace(line) == ".turbo" {
+			hasTurbo = true
+		}
+	}
+
+	if !hasTurbo {
+		gitignore, err := os.OpenFile(".gitignore", os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
As with above, we want to ensure we don't have an implicit dependency on where `turbo` was invoked. You can use `l.base.RepoRoot.Join(".gitignore").OpenFile`

See functions defined [here](https://github.com/vercel/turborepo/blob/main/cli/internal/turbopath/absolute_path.go#L83)